README.md

```diff
@@ -199,108 +102,79 @@ $ sudo ./setup_host_rpi.sh
 $ sudo reboot
 ```
 
+Optionally for portable use, you can configure the Pi to prioritize your mobile hotspot as follows:
+
 After, you will need to run the `setup_hotspot.py` script: 
 ```bash
-# Transfer setup_hotspot.py to Raspberry Pi
-$ scp /path/to/nocturne-image/setup-scripts/setup_hotspot.py pi@raspberrypi.local:/home/pi/
-
 # SSH into Raspberry Pi
 $ ssh pi@raspberrypi.local
 
+# Download setup_hotspot.py to Raspberry Pi
+$ wget https://raw.githubusercontent.com/usenocturne/nocturne-image/refs/heads/main/setup-scripts/setup_hotspot.py
```

Comment:
Remember that this would need to be updated to `setup_hotspot_rpi.py` because of the rename.

Reply:
The comment also needs to be renamed, but otherwise this is fixed.